package.json

```diff
@@ -17,7 +21,7 @@
     "docker:build": "docker build -t opentermsarchive.org . --build-arg ENV_FILE=.env",
     "docker:run": "(docker rm opentermsarchive.org || true) && docker run --rm -p 3000:3000 --name opentermsarchive.org -it opentermsarchive.org",
     "docker:ssh": "docker exec -it opentermsarchive.org /bin/sh",
-    "test": "jest",
+    "test:links": "linkinator http://localhost:$npm_package_config_port --recurse --skip https://www.linkedin.com/.* --verbosity error",
```

Comment:
> This is just for your information, as a way to discover what I believe to be a cool hidden gem, I don't believe it is needed:
```suggestion
    "pretest": "npm start &",
    "test": "linkinator http://localhost:$npm_package_config_port --recurse --skip https://www.linkedin.com/.* --verbosity error",
    "posttest": "npm stop"
```

> This means we actually have a `test` procedure that is part of the NPM lifecycle scripts (i.e. the `npm test` command is very well-known and works, so it can integrate with any system without configuring the name of the script to run); since we need the server to run for these tests, instead of adding complexity in the `test` command itself, we use the `pre` and `post` hooks in the NPM scripts to make it readable and delegate ordering and error management to NPM itself. The `test` command itself stays focused 🙂

Reply:
Thanks for these explanations, it is very useful to learn these hooks.